mathics/builtin/inout.py

```diff
@@ -385,7 +385,7 @@ class ToBoxes(Builtin):
 
     Unlike 'MakeBoxes', 'ToBoxes' evaluates its argument:
     >> ToBoxes[a + a]
-     = RowBox[{2, \u2062, a}]
+     = RowBox[{2,  , a}]
```

Comment:
I thought this might be related to the web frontend but actually it looks better with the space I think.